components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -211,6 +219,9 @@ interface KeyringController {
   AddObserver(pending_remote<KeyringControllerObserver> observer);
   SetDefaultKeyringDerivedAccountName(string address, string name) => (bool success);
   SetDefaultKeyringImportedAccountName(string address, string name) => (bool success);
+  AddHardwareAccounts(array<HardwareWalletAccount> info);
+  GetHardwareAccounts() => (array<HardwareWalletAccount> accounts);
```

Comment:
we should return `array<AccountInfo>` so UI can just concat the result to `accountInfos` in `KeyringInfo`.
And probably rename this API to be `GetHardwareAccountInfos`

Reply:
done